### DISCUSSION TITLE
如何控制音频的语速

### DISCUSSION THREAD

**Discussion by matyhtf** [lenML/Speech-AI-Forge#15] [问答 | Q&A] [answered]
默认生成的音频，语速较快，是否可以通过参数来调整语速？

**Comment by zhzLuke96** [comment, marked as answer]
简单说有两个方法，prompt，或者调用api使用原始的音频加速，具体看你要用在哪

1. 使用prompt调整语速：
如果你在使用 webui 入口，在tts tab左下角有 `Prompt engineering` 模块
你可以在 `prefix` 栏填入 `[speed_5]` 以调整语速，最高 [speed_9] 最低 [speed_0] 没有写的话模型以文本内容自行推断也可能是随机
（虽然即使写了 prefix 也有可能不会变快，但是大概率是会有效果）

2. 调用接口
具体部署之后看 /docs 下，有 speed 参数的接口都可以调整
当然，调用接口也可以写 prefix

**Reply by matyhtf** [reply]
谢谢回复，我试试

**Comment by alex9441** [comment]
ChatTTS的默认语速比较快,有的应用场景下 [speed_0] 的语速都显得过快.
